bors.toml

```diff
@@ -0,0 +1,21 @@
+status = [
+    "ci/gitlab/gitlab.parity.io",
+    "continuous-integration/gitlab-build-wasm",
+    "continuous-integration/gitlab-check-std",
+    "continuous-integration/gitlab-check-wasm",
+    "continuous-integration/gitlab-clippy-std",
+    "continuous-integration/gitlab-clippy-wasm",
+    "continuous-integration/gitlab-examples-clippy-std",
+    "continuous-integration/gitlab-examples-clippy-wasm",
+    "continuous-integration/gitlab-examples-contract-build",
+    "continuous-integration/gitlab-examples-fmt",
+    "continuous-integration/gitlab-examples-generate-metadata",
+    "continuous-integration/gitlab-examples-test",
+    "continuous-integration/gitlab-fmt",
+    "continuous-integration/gitlab-publish-docs",
+    "continuous-integration/gitlab-test"
+]
+# The default is one hour.
+timeout_sec = 600
```

Comment:
But `600` seconds are 10 minutes, `3600` seconds are an hour.. I am fine with that though but either comment or number needs correction.

Reply:
that means that it will default when not set.